python/feldera/__init__.py

```diff
@@ -1,3 +1,71 @@
+"""
+This package provides a simple interface for working with feldera via python.
+
+You will need to first import `FelderaClient`, and give it the URL, api_key (if required)
+of the feldera instance you want to connect to.
+
+You can then use `SQLContext` to set up the source (something that provides data to feldera) and the sink
+(something that receives data from feldera).
+
+You use SQL to write the query that will be executed on the data coming from the source. This query creates a
+"view" in feldera. You can forward the data from this view to the sink of your choice, or stack views on
+top of each other.
```

Comment:
```suggestion
* Create an `SQLContext` object and add SQL tables, which specify inputs to Feldera, and views, which define queries to run on these inputs.
* Set up sources, which provide data to tables, and sinks, which consume data from views
* Call `start` on the `SQLContext` object to create a Feldera pipeline to continuously ingest data from sources into SQL tables, incrementally evaluate SQL views, and send outputs to sinks. Alternatively, call `run_to_completion` to create a pipeline that processes all inputs to completion and exits.
```

I don't know how sphinx works, but it would be nice to add links to corresponding doc sections above.